airflow/hooks/dbapi_hook.py

```diff
@@ -61,10 +61,17 @@ def __init__(self, *args, **kwargs):
             raise AirflowException("conn_name_attr is not defined")
         elif len(args) == 1:
             setattr(self, self.conn_name_attr, args[0])
-        elif self.conn_name_attr not in kwargs:
-            setattr(self, self.conn_name_attr, self.default_conn_name)
-        else:
+        elif self.conn_name_attr in kwargs:
             setattr(self, self.conn_name_attr, kwargs[self.conn_name_attr])
+        else:
+            setattr(self, self.conn_name_attr, self.default_conn_name)
+
+    @property
+    def conn_name(self) -> str:
+        """
+        Get conn_name to avoid pylint no-member in subclass
```

Comment:
We don't add production code only because a linter is unhappy!

What it give us is that it..
```suggestion
        Get the name of the database connection attribute which identifies the connection.
```

Reply:
You're right

Reply:
You're right.

Reply:
You missed that.

Reply:
Sorry, I just force push and forget fetch from my origin